examples/agents/human_in_the_loop/cdk/BedrockAgentStack/BedrockAgentStack_stack.py

```diff
@@ -54,6 +54,32 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             )
         )
 
+        base_lambda_policy = iam.ManagedPolicy(
+            self,
+            "LambdaBasicExecutionPolicy",
+            description="Allows Lambda functions to write to CloudWatch Logs",
+            statements=[
+                iam.PolicyStatement(
+                    effect=iam.Effect.ALLOW,
+                    actions=[
+                        "logs:CreateLogGroup",
+                        "logs:CreateLogStream",
+                        "logs:PutLogEvents"
+                    ],
+                    resources=[
+                        f"arn:aws:logs:{self.region}:{self.account}:log-group:/aws/lambda/BedrockActionGroup-HRAssistant"
```

Comment:
Can you create the log group ahead of time and reference it here? That would also let you set an expiration on the log group (instead of the default of never expire)